test/e2e/storage/persistent_volumes-local.go

```diff
@@ -1059,6 +1087,30 @@ func createLocalPod(config *localTestConfig, volume *localTestVolume) (*v1.Pod,
 	return framework.CreateSecPod(config.client, config.ns, []*v1.PersistentVolumeClaim{volume.pvc}, false, "", false, false, selinuxLabel)
 }
 
+func createLocalPodWithFsGroup(config *localTestConfig, volume *localTestVolume, fsGroup *int64) (*v1.Pod, error) {
```

Comment:
`MakeSecPod` already sets an fsgroup, so we shouldn't need this

Reply:
@msau42 
`MakeSecPod` use a fixed `fsGroup` to create pod, but I need to different `fsGroup` for each pod. I cannot use it. So I need this function.

Another way is to extend `MakeSecPod` and `CreateSecPod` to accept `fsGroup` parameter. Then I can reuse `MakeSecPod`. What do you think?

Reply:
Yeah, reusing is fine.

Reply:
@msau42 Done in https://github.com/kubernetes/kubernetes/pull/62102/commits/2b3bdd1f6ba36b84962f30f37dc21e7d70c26f98.